sdk/machinelearningservices/Azure.ResourceManager.MachineLearning/src/Generated/Models/EndpointServiceConnectionStatus.cs

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.ResourceManager.MachineLearning.Models
+{
+    /// <summary> Connection status of the service consumer with the service provider. </summary>
+    public readonly partial struct EndpointServiceConnectionStatus : IEquatable<EndpointServiceConnectionStatus>
```

Comment:
This enum is different from `MachineLearningPrivateEndpointServiceConnectionStatus` in that it doesn't have `Timeout` value. Just want to confirm that it's expected.

Reply:
MachineLearningPrivateEndpointServiceConnectionStatus is part of SharedPrivateLinkResourceProperty https://github.com/Azure/azure-rest-api-specs/blob/main/specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2024-04-01/machineLearningServices.json

EndpointServiceConnectionStatus is part of RegistryPrivateLinkServiceConnectionState https://github.com/Azure/azure-rest-api-specs/blob/main/specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2024-04-01/registries.json